tests/test_bonded.py

```diff
@@ -1,31 +1,37 @@
 import numpy as np
 import jax
-import jax.numpy as jnp
-from jax.config import config; config.update("jax_enable_x64", True)
+from jax.config import config
+
+config.update("jax_enable_x64", True)
 import functools
 
 from common import GradientTest
 from timemachine.lib import potentials
 from timemachine.potentials import bonded
 
+relative_tolerance_at_precision = {
```

Comment:
In general, at least for nonbonded forces, we probably can't do this. Different forces can achieve different levels of precision depending on the dynamic range. We generally want it to be as tight as possible for a particular force.

Reply:
That makes sense -- so if we were to have a single definition somewhere, rtol would depend on both `force_name` and `fp_precision`, not just `fp_precision` as I had encoded here

Reply:
Wouldn't it be better then to just have the various precision/tolerances local to the function itself (as how it used to be) as opposed to having to scroll up and down to see what they are?

Reply:
I'll revert this change -- I had noticed that the same (precision, rtol) pairs were used for all forces I had looked at in this file, but since they will in general be different across files and force types, there's not much sense in extracting the definition.

Reply:
Resolved in https://github.com/proteneer/timemachine/pull/299/commits/72decf1a0194c811d528bb1f7caf1f00fdda5db6